apps/web/src/app/(private)/projects/[projectId]/versions/[commitUuid]/preview/@modal/page.tsx

```diff
@@ -0,0 +1,3 @@
+export default function ModalPage() {
+  return null
+}
```

Comment:
TIL [parallel routes + route intercepting](https://nextjs.org/docs/app/api-reference/file-conventions/parallel-routes#modals) to do modals in routes. We typically do those in Layout components but those have the drawback of not having access to search params. This alternative method allows us to keep UI in page components (with proper server-side search params access) and keep modals in their own routes.

The DX is terrible but the end results works ok.